pymc3/distributions/multivariate.py

```diff
@@ -505,7 +505,13 @@ def __init__(self, n, p, *args, **kwargs):
     def _random(self, n, p, size=None):
         if size == p.shape:
             size = None
-        return np.random.multinomial(n, p, size=size)
+
+        if p.ndim > 1 and p.shape[0] > 1:
```

Comment:
it seems like there are two cases we're willing to support: 1-d or 2-d probability matrices.  I think it is fair to use
```
if p.ndim == 1:
    ...
elif p.ndim == 2:
   ...
else:
    raise ValueError('Outcome probabilities must be 1- or 2-dimensional (supplied `p` has {} dimensions)'.format(p.ndim))
```